src/Colog/Core/Action.hs

```diff
@@ -6,9 +6,12 @@
 {-# LANGUAGE UndecidableInstances #-}
 
 {- |
-Copyright:  (c) 2018-2020 Kowainik
-SPDX-License-Identifier: MPL-2.0
-Maintainer: Kowainik <xrom.xkov@gmail.com>
+Module                  : Colog.Core.Action
+Copyright               : (c) 2018-2020 Kowainik
```

Comment:
Currently, the `Copyright` field is not updated as I would like to. I've created a corresponding issue in `headroom`:

* https://github.com/vaclavsvejcar/headroom/issues/79